tests/test_compatibility.py

```diff
@@ -243,3 +242,30 @@ def test_signer(implementations):
 
     assert _verify_message(umbral1, pk_bytes, signature2_bytes, message)
     assert _verify_message(umbral2, pk_bytes, signature1_bytes, message)
+
+
+def _measure_sizes(umbral):
+
+    sized_types = [
+        umbral.SecretKey,
+        umbral.SecretKeyFactory,
+        umbral.PublicKey,
+        umbral.Signature,
+        umbral.Capsule,
+        umbral.KeyFrag,
+        umbral.VerifiedKeyFrag,
+        umbral.CapsuleFrag,
+        umbral.VerifiedCapsuleFrag,
+        ]
+
+    return {tp.__name__: tp.serialized_size() for tp in sized_types}
```

Comment:
Does this output some like:
```
{
   "SecretKey": 17
   ...
}
```?

Reply:
Yep. Can't just put a class there, because it will be different for the other implementation